src/components/pod/pod.stories.mdx

```diff
@@ -4,16 +4,236 @@ import Heading from '../heading';
 import Button from '../button';
 
 <Meta
-  title="Design System/Pod"
+  title="Pod"
   parameters={{ info: { disable: true }}}
 />
 
+# Pod
+Presents content grouped visually together. This can be text, or other components. A good example is a ‘tile’ showing contact information for an individual.
+
+Configure Pod and Fieldset components to work together, or choose the pre-configured Show/Edit Pod component:
+
+The ShowEditPod component automatically presents content as a read-only Pod, until the user clicks an edit icon, which shows the same information in an editable Fieldset.
+
+But, configuring Pod and Fieldset components manually will give you more customization options, like the following.
```

Comment:
Starting a sentence with a conjunction like 'but' is usually discouraged in English, try 'however' instead.
```suggestion
The ShowEditPod component automatically presents content as a read-only Pod, until the user clicks an edit icon, which shows the same information in an editable Fieldset. However, configuring Pod and Fieldset components manually will give you more customisation options, like the following:
```